Cargo.toml

```diff
@@ -333,3 +334,7 @@ inherits = "release"
 lto = "fat"
 # https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units
 codegen-units = 1
+
+[patch.crates-io]
```

Comment:
Is this section really needed here? `sp-io` is already found at "./primitives/io". And `sp-arkworks` dependency simply needs path specified in `primitives/io/Cargo.toml`